pandas/tests/io/formats/test_to_html.py

```diff
@@ -1905,6 +2013,20 @@ def test_to_html_multiindex_max_cols(self):
         </table>""")
         assert result == expected
 
+    @pytest.mark.parametrize('header', [True])
+    @pytest.mark.parametrize('index', [True])
+    @pytest.mark.parametrize('index_names', [True])
+    def test_to_html_index_names(self, expected_output,
+                                 idx_type, col_idx_type, index, header,
+                                 index_names):
+        df = DataFrame(np.zeros((2, 2), dtype=int),
+                       index=idx_type, columns=col_idx_type)
+        result = df.to_html(index=index, header=header,
+                            index_names=index_names)
+        assert result == expected_output(idx_type, col_idx_type,
```

Comment:
Can you do `expected = expected_output(...` on the line preceding this then change the assertion to simply `result == expected`?

Reply:
no problem. will apply this change to #22655.